BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/Internal/RTTComms.cs

```diff
@@ -400,6 +407,7 @@ private void onRecv(string in_message)
             {
                 m_clientRef.Log("RTT RECV: " + in_message);
             }
+            m_disconnectedWithReason = false;
```

Comment:
Not sure why you set it to false here, but I would like you to set it to false in `EnableRTT`.